### ISSUE TITLE
Majority robs minority

### ISSUE THREAD

**Issue by CJentzsch** [ares-protocol/ares-contracts#1]
If a malicious proposal (the majority of the DAO wants to rob the minority) is going to be accepted, all the minority can do is withdraw the remaining amount of ether.
Future profits (in ETH or in ERC20 tokens) paid by funded projects, will only go to the malicious majority.
So game theoretically it is very profiting for that majority, to first invest in good projects which will pay dividends in the future, and then make the minority leave with a fraction of their initial investment.

I would suggest a "reward fund", were ARES DAO token owners will get a fraction of (Reward/Dividend tokens?), which they will keep when they are leaving.

**Comment by SilentCicero** [comment]
@CJentzsch agreed, I was going to craft a complex fund for ERC20 so proportional share tokens can be withdrawn. You beat me to it =D
